CHANGELOG.md

```diff
@@ -368,6 +368,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - The strategies that support `sync_batchnorm` now only apply it when fitting ([#11919](https://github.com/PyTorchLightning/pytorch-lightning/pull/11919))
 
 
+- Changed `Trainer.data_parallel_device_ids` to return a list of GPU device indexes ([#12072](https://github.com/PyTorchLightning/pytorch-lightning/pull/12072))
+
```

Comment:
```suggestion
```
The trainer arg remains unchanged, only deprecated.

Reply:
Removed